### DISCUSSION TITLE
Documentation site Algolia search not working on mobile view

### DISCUSSION THREAD

**Discussion by mhobesong** [twbs/bootstrap#36955] [General]
The search Algolia input field does not work while on mobile view. I noticed this on the documentation page: https://getbootstrap.com/docs/5.2/getting-started/introduction/

<img width="863" alt="image" src="https://user-images.githubusercontent.com/5602093/184619310-7a16a18b-e397-422e-a91e-d48b0bed13db.png">

**Comment by julien-deramond** [comment]
Hey @mhobesong. It's referenced in https://github.com/twbs/bootstrap/issues/36822. The fix will be released in the v5.2.1.
